### PR TITLE
Adding docs for MS SQL colletor/analyzer.

### PR DESCRIPTION
**Pull request by chasehainey**
Added two new md files for mssql collection and analysis.

### CODE REVIEW

File: docs/source/analyze/mssql.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: MS SQL
+description: Check version and connection status
+---
+
+The `MS SQL` analyzer is available to check vesion and connection status of a PostgreSQL database.
```
Comment (banjoh) [inline docs/source/analyze/mssql.md:6]: ...of a **Microsoft SQL Server** database...

File: docs/source/analyze/mssql.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+The `when` value in an outcome of this analyzer contains the connection or version information.
+
+The conditional in the when value supports the following:
+
+**connected:** A boolean representing whether the database is connected.
+Can be compared to a boolean value with the `==` operator.
+
+**version:** A string representing the version of the database.
+Can be compared to a semver string using `<`, `<=`, `>`, `>=`, `==`, `!=`, with the letter 'x' as a version wildcard (10.x).
```
Comment (banjoh) [inline docs/source/analyze/mssql.md:27]: ...to **an assembly version** string using...

I'm referring to Microsoft's assembly version format => https://learn.microsoft.com/en-us/dotnet/standard/assembly/versioning#assembly-version-number

File: docs/source/collect/mssql.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Included resources
+
+A single JSON file will be added to the support bundle, in the path `/mssql/[collector-name].json`:
+
+```json
+{
+    "isConnected": false,
+    "error": "invalid password",
+    "version": "10.12",
```
Comment (banjoh) [inline docs/source/collect/mssql.md:43]: "version": "15.0.2000.1565"

Let's bring closer to reality

**Review by chasehainey — COMMENTED** [review]
Thanks for reviewing and catching those erroneous artifacts from Postgres.

File: docs/source/collect/mssql.md (hunk 3 above)
Comment (chasehainey) [reply docs/source/collect/mssql.md]: Fixed. Appreciate the review.

File: docs/source/analyze/mssql.md (hunk 2 above)
Comment (chasehainey) [reply docs/source/analyze/mssql.md]: Fixed. Appreciate the review.

File: docs/source/analyze/mssql.md (hunk 1 above)
Comment (chasehainey) [reply docs/source/analyze/mssql.md]: Fixed. Appreciate the review.

File: docs/source/analyze/mssql.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ The conditional in the when value supports the following:
 Can be compared to a boolean value with the `==` operator.
 
 **version:** A string representing the version of the database.
-Can be compared to a semver string using `<`, `<=`, `>`, `>=`, `==`, `!=`, with the letter 'x' as a version wildcard (10.x).
+Can be compared to a an assembly version string using `<`, `<=`, `>`, `>=`, `==`, `!=`, with the letter 'x' as a version wildcard (10.x).
```
Comment (banjoh) [inline docs/source/analyze/mssql.md:27]: Extra "a" in `compared to a an assembly version` :)

**Review by banjoh — APPROVED** [review]
Looks good

**Pull request merged**